internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -46,11 +46,14 @@ const (
 	handlerContextKey ctxKey = "StorageHandler"
 	configPath               = "res/configuration.toml"
 	numberOfReadings         = "readingCount"
-
-	apiResourceRoute           = clients.ApiBase + "/device/{" + deviceNameKey + "}/resource/{" + resourceNameKey + "}"
-	apiResourceRouteMultiple   = clients.ApiBase + "/device/{" + deviceNameKey + "}/resource/{" + resourceNameKey + "}/{" + numberOfReadings + "}"
-	apiWithoutResRoute         = clients.ApiBase + "/resource/{" + resourceNameKey + "}"
-	apiWithoutResRouteMultiple = clients.ApiBase + "/resource/{" + resourceNameKey + "}/{" + numberOfReadings + "}"
+	startKey                 = "start"
+	endKey                   = "end"
```

Comment:
How about keeping keys together? I'd like to suggest to move start and end keys after resourceNameKey.

Reply:
Done!